File: CUDC/CUDC.Client/src/app/models/survey/survey-info.ts
import { BasicInfo } from '../search/basic-info';
import { Question } from './question';
import { QuestionOption } from './question-option';
import { QuestionReference } from './question-reference';
import { Response } from './response';
import { Section } from './section';
import { Survey } from './survey';

export class SurveyInfo {
    basicInfo: BasicInfo;
    survey: Survey;
    sections: Section[];
    questions: Question[];
    questionOptions: QuestionOption[];
    questionReferences: QuestionReference[];
    answers: Response;
    hasPreSubmitedSurvey: Boolean;
}

File: CUDC/CUDC.Common/Dtos/SurveyInfo.cs
using CUDC.Common.Dtos.CuSearch;
using System;
using System.Collections.Generic;

namespace CUDC.Common.Dtos
{
    public class SurveyInfo
    {
        public BasicInfo BasicInfo { get; set; }

        public SurveyDto Survey { get; set; }

        public IEnumerable<SectionDto> Sections { get; set; }

        public IEnumerable<QuestionDto> Questions { get; set; }

        public IEnumerable<QuestionOptionDto> QuestionOptions { get; set; }

        public ResponseDto Answers { get; set; }

        public IEnumerable<QuestionReferenceDto> QuestionReferences { get; set; }

        public Boolean HasPreSubmitedSurvey { get; set; }

    }
}
